tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -2150,7 +2236,7 @@ public:
    /// for (auto &&colName : colNames) std::cout << colName << std::endl;
    /// ~~~
    ///
-   ColumnNames_t GetColumnNames()
+   ColumnNames_t GetColumnNames() const
```

Comment:
this is unrelated to the PR, and should be discussed carefully: it's a change in RDataFrame's public interface, and a `const` method promises to be thread-safe, which should be checked. Please remove for now.

Reply:
I will remove the `const` (here and on the methods below), but note that in this case `Describe()` can no longer be `const`.